include/s3select.h

```diff
@@ -2122,15 +2160,17 @@ class csv_object : public base_s3object
     m_extract_csv_header_info(false),
     m_previous_line(false),
     m_skip_first_line(false),
-    m_processed_bytes(0) {}
+    m_processed_bytes(0),
+    m_processed_rows(0) {}
```

Comment:
`m_processed_rows` should be handled in `base_s3object`
as part of the unified execution flow (it should work for all data sources CSV, JSON, and Parquet)